docs/source/terms_and_model.rst

```diff
@@ -339,194 +287,144 @@ A contiguous region specified by chromosomal bands features.
-  Precise :ref:`SequenceLocation` should be preferred when
-  nucleotide-scale location is known.
-* `species` is specified using the NCBI taxonomy.  The CURIE prefix
-  MUST be "taxonomy", corresponding to the `NCBI taxonomy prefix at
-  identifiers.org
-  <https://registry.identifiers.org/registry/taxonomy>`__, and the
-  CURIE reference MUST be an NCBI taxonomy identifier (e.g., 9606 for
-  Homo sapiens).
-* ChromosomeLocation is intended primarily for Humans.  Support for
-  other species is possible and will be considered based on community
-  feedback.
-* `chromosome` is an archetypal chromosome name. Valid values for, and
-  the syntactic structure of, `chromosome` depends on the species.
-  `chromosome` MUST be an official sequence name from `NCBI Assembly
-  <https://www.ncbi.nlm.nih.gov/assembly>`__.  For Humans, valid
-  chromosome names are 1..22, X, Y (case-sensitive).
-* `interval` refers to a contiguous region specified named markers,
-  which are presumed to exist on the specified chromosome.  See
-  :ref:`CytobandInterval` for additional information.
-* The conversion of ChromosomeLocation instances to SequenceLocation
-  instances is out-of-scope for VRS.  When converting `start` and
-  `end` to SequenceLocations, the positions MUST be interpreted as
-  inclusive ranges that cover the maximal extent of the region.
-* Data for converting cytogenetic bands to precise sequence
-  coordinates are available at `NCBI GDP
-  <https://ftp.ncbi.nlm.nih.gov/pub/gdp/>`__, `UCSC GRCh37 (hg19)
-  <http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/cytoBand.txt.gz>`__,
-  `UCSC GRCh38 (hg38)
-  <http://hgdownload.cse.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz>`__,
-  and `bioutils (Python)
-  <https://bioutils.readthedocs.io/en/stable/reference/bioutils.cytobands.html>`__.
-* See also the rationale
-  for :ref:`dd-not-using-external-chromosome-declarations`.
+.. _SystemicVariation:
 
+Systemic Variation
+##################
 
-**Examples**
+Systemic Variation is a :ref:`Variation` of multiple
+molecules in the context of a system, e.g. a genome,
+sample, or homologous chromosomes.
 
-.. parsed-literal::
+.. _Abundance:
 
-   {
-     "chr": "11",
-     "interval": {
-       "end": "q22.3",
-       "start": "q22.2",
-       "type": "CytobandInterval"
-       },
-     "species_id": "taxonomy:9606",
-     "type": "ChromosomeLocation"
-   }
+Abundance
+$$$$$$$$$
 
-.. _sequence-location:
-.. _sequencelocation:
+*Abundance* is the measure of a quantity of a molecule
+in a system. :ref:`Copy Number <CopyNumber>` and
+gene expression variants are two common types of
+abundance variation, measuring the copies of a molecule
+present in a genome or expressed in a sample, respectively.
 
-SequenceLocation
-$$$$$$$$$$$$$$$$
+.. _CopyNumber:
 
-**Biological Definition**
+CopyNumber
+%%%%%%%%%%
 
-A specified subsequence within another sequence that is used as a reference sequence.
+*Copy Number* captures the copies of a molecule within
```

Comment:
change to 
`a molecule or molecular region`

Reply:
something like this. Talk about variation and features.